packages/core/strapi/lib/middlewares/index.js

```diff
@@ -12,14 +12,14 @@ const query = require('./query');
 const responseTime = require('./response-time');
 const responses = require('./responses');
 const security = require('./security');
-// TODO: add back ?
-// session: require('./session'),
+const session = require('./session');
 const publicStatic = require('./public');
 
 module.exports = {
   errors,
   ip,
   security,
+  session,
```

Comment:
This would be a breaking change correct @alexandrebodin ? It would need to be added to the default `./config/middlewares.js`?